src/pages/GithubApp/Components/Navbar.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import useDarkMode from '../../../hook/useDarkMode';
+
+function Navbar() {
+  const [colorTheme, setTheme] = useDarkMode();
+  return (
+    <div className="group flex justify-between items-center ">
+      <h1 className="dark:text-white text-2xl font-[900]transition duration-300">
+        devifinder
+      </h1>
+      <div className="flex justify-between items-center w-[90px]">
+        {colorTheme === 'light' ? (
+          <>
+            <p className="dark:text-gLightLightGray text-gLightDarkBlue font-[900] tracking-widest group-hover:text-gLightDarkBlue transition duration-300">
```

Comment:
Your only difference between this class list and the other one is group-hover, as well as the text and the SVG. Do use an array here, something like:
```js
[
  {svg: svgLight, text: 'Light', color: 'gLightDarkBlue'},
  {...}//dark props
].map((buttonProps) => (
  <p className=`..... group-hover:text-${buttonProps.color}...`>
    {buttonProps.text}
    {buttonProps.svg}
  </p>
)
```